### ISSUE TITLE
Difference between dates is counter-intuitive

### ISSUE THREAD

**Issue by jbgi** [johanandren/timeforscala#15]
Currently
```scala
scala> val d = LocalDateTime.now
d: java.time.LocalDateTime = 2018-04-27T15:17:34.548

scala> d + Seconds(3) - d == Seconds(3)
res1: Boolean = false

scala> d + Seconds(3) - d == Seconds(-3)
res2: Boolean = true
```
I would have expected the opposite.

Can we change that?

**Comment by johanandren** [comment]
Hmm. I agree, it looks backwards.

**Comment by johanandren** [comment]
Released 1.7 just now
